test_plans/ceph-with-dash-lxd.yaml

```diff
@@ -0,0 +1,5 @@
+bundle: bundle:~juju-qa/ceph-with-dash
+bundle_name: ceph-with-dash-lxd
+bundle_file: ceph-with-dash-lxd.yaml
```

Comment:
This is bad, because  bundles are allowed to have multiple bundle files. many do. I made a new bundle file so that we can work with upstream. Each bundle has a bundle.yaml. if we do a do a fork and destroy upstream's files, we are committing QA staff to own this. I am sure